homeassistant/components/websocket_api/commands.py

```diff
@@ -303,7 +303,9 @@ async def handle_render_template(hass, connection, msg):
 
     if timeout:
         try:
-            timed_out = await template_obj.async_render_will_timeout(timeout)
+            timed_out = await template_obj.async_render_will_timeout(
```

Comment:
I think that we should allow passing in `strict`. Right now you're updating this WS command and it is used in many other places, like Markdown cards etc.

Reply:
So make it optional and default it to false.